exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello World!"
+console.log(greeting);
+console.log(greeting);
 console.log(greeting);
```

Comment:
The above lines work well.
You could also combine multiple console logs like this:
`console.log(greeting, greeting, greeting)`